scripts/release-notes.js

```diff
@@ -0,0 +1,126 @@
+const {
+  exec,
+  releaseYargsBuilder,
+  requireGitHubCLI,
+  buildQuery,
+} = require('./release-utils');
+const diffFeatures = require('./diff-features');
+
+function main(argv) {
+  const { startVersionTag, endVersionTag } = argv;
+
+  requireGitHubCLI();
+
+  const allAdds = [];
+  const allRemoves = [];
+
+  console.error(
```

Comment:
I noticed you're using console errors vs. console logs throughout this script, any particular reason why?

Reply:
I've used `console.log()` for the actual release notes output (see lines 51-53), but all the status messages (and actual errors) are written to stderr to facilitate redirection. For instance, you could run `node ./scripts/release-notes.js | pbcopy` (on macOS) to copy the output directly the clipboard, without cluttering the results with status messages.

Reply:
Gotcha, that makes sense!